src/Compilers/CSharp/Portable/Symbols/AliasSymbol.cs

```diff
@@ -48,22 +48,30 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal abstract class AliasSymbol : Symbol
     {
         private readonly ImmutableArray<Location> _locations;  // NOTE: can be empty for the "global" alias.
+        private readonly string _aliasName;
+        private readonly bool _isExtern;
+        private readonly Symbol? _containingSymbol;
```

Comment:
>Symbol? _containingSymbol; [](start = 25, length = 26)

Can the containing symbol be `null`?
Documentation on `ContainingSymbol` below may be out-of-date (sounds like it wouldn't be null)

Reply:
> Can the containing symbol be null?

The refactoring doesn't make any changes in that regard.

---
In reply to: [588799470](https://github.com/dotnet/roslyn/pull/51676#discussion_r588799470) [](ancestors = 588799470)